adapters/audienceNetwork/audienceNetworktest/exemplary/banner.json

```diff
@@ -43,9 +43,6 @@
                     "Accept": [
                         "application/json"
                     ],
-                    "Auth-Token": [
-                        "test-app-id|test-app-secret"
-                    ],
                     "Content-Type": [
                         "application/json;charset=utf-8"
                     ],
```

Comment:
The `"Auth-Token"` is gone but, from the PR title, should we expect to have an `"authentication_id"` field added inside body as shown below? I don't see that addition in the PR diff, is this field already there or will it be implemented later?
```
39 {
40     "expectedRequest": {
41         "uri": "https://an.facebook.com/placementbid.ortb",
42         "headers": {
43             "Accept": [
44                 "application/json"
45             ],
46   -         "Auth-Token": [
47   -             "test-app-id|test-app-secret"
48   -         ],
49             "Content-Type": [
50                "application/json;charset=utf-8"
51            ],
52            "X-Fb-Pool-Routing-Token": [
53                "v4_bidder_token"
54            ]
55        },
56        "body": {
57 +          "authentication_id": "some-id-here"
58            "id": "test-req-id",
59            "imp": [
adapters/audienceNetwork/audienceNetworktest/exemplary/banner.json
```

Reply:
@guscarreon `authentication_id` already exists is the request body. In this file it's here: https://github.com/prebid/prebid-server/blob/2508d1c1c113ed751153e9e7ddfe7211800ac64b/adapters/audienceNetwork/audienceNetworktest/exemplary/banner.json#L87

Reply:
Ohhh I completely missed it Mansi. I looked for it in the codebase but probably probably mistyped it and that's the reason I didn't find it. Pretty good catch